### DISCUSSION TITLE
Implementing segment anything geospatial beyond RGB.

### DISCUSSION THREAD

**Discussion by lkpanganiban** [opengeos/segment-geospatial#241] [Q&A] [answered]
Any guidance on how to implement segment anything beyond 3 bands? I am thinking of implementing a SAM with RGB + Elevation values. This will greatly improve the accuracy of the segmentation since you have a depth value of each pixel.

**Comment by giswqs** [comment, marked as answer]
Not supported at thge moment. This is the limitation of the original Segment Anything Model.
